packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2115,6 +2141,7 @@ export const components = ({
 
           ...(ownerState.disabled && {
             color: odysseyTokens.TypographyColorDisabled,
+            opacity: "1 !important",
```

Comment:
Can we avoid using `!important`? This might be a MUI restriction, but `!important` can always create a lot of issues especially when we're theming.

Reply:
If we can't avoid `!important`, let's add a comment above.

Reply:
The only way to avoid it is to target the element using the `Mui-disabled` class, rather than the `ownerState`